automodel.py

```diff
@@ -23,6 +23,7 @@ def completion_tensors(
             output = self.model.generate(
                 **inputs,
                 do_sample=True,
+                use_cache=True,
```

Comment:
Does this create warnings for every generation? I ask because this is deprecated. @cassanof

Reply:
I'm not sure if it does, but this is handy when models don't have it turned on in the generation config. Should I remove it?